src/timeranges/_timeranges.py

```diff
@@ -94,6 +94,8 @@ def __attrs_post_init__(self) -> None:
         self.validate()
 
     def contains(self, t: time) -> bool:
+        if self.time_ranges == []:
```

Comment:
It'd be better to use `if not self.time_ranges`. The "boolean" equivalent of lists is whether or not it has things inside, so if it has things, `bool(my_list) == True`, and it's `False` if it's empty.